utils/gear-replay-cli/README.md

```diff
@@ -34,63 +33,60 @@ Other than that it's a good tool.
 
 ### Usage examples
 
-* Execute block on Vara live chain
-
-    * current latest finalized block on Vara chain
-
-        ```bash
-        gear try-runtime --chain=vara --runtime vara_runtime.compact.compressed.wasm execute-block live --uri wss://archive-rpc.vara-network.io:443
-        ```
+- Execute block on Vara live chain
 
-    * at block `$HASH`
+  - current latest finalized block on Vara chain
```

Comment:
plz verify the modifications in this file @ekovalev, these changes are auto applied by running 

```
# cargo install typos-cli
typos -w
```

Reply:
I think `-` vs. `*` doesn't really matter as long as the markdown file adheres to the spec. If that's the only change then it's fine.